dev/bots/prepare_package.dart

```diff
@@ -39,153 +32,212 @@ class ProcessFailedException extends Error {
 
 /// Creates a pre-populated Flutter archive from a git repo.
 class ArchiveCreator {
-  /// [tempDir] is the directory to use for creating the archive.  Will place
-  /// several GiB of data there, so it should have available space.
-  /// [outputFile] is the name of the output archive. It should end in either
-  /// ".tar.xz" or ".zip".
-  /// The runner argument is used to inject a mock of [Process.runSync] for
+  /// [_tempDir] is the directory to use for creating the archive.  The script
+  /// will place several GiB of data there, so it should have available space.
+  ///
+  /// The starter argument is used to inject a mock of [Process.start] for
```

Comment:
This doc comment probably needs an update since `[Process.start]` is no more.

Reply:
And while you're in here: maybe document `subprocessOutput`?

Reply:
Done